File: src/Model/Villager.java
package Model;

import Technical.PowerEnum;

public class Villager extends GameCharacter {

    public Villager() {
        super("Villager",
                "The villager has no other power than its own deduction to find out the werewolves.",
                false,
                PowerEnum.Constant,
                "../../img/Thiercelieux.png");
    }
}

File: app/src/main/java/Model/Villager.kt
package Model

class Villager(order: Int)
    : Character("The villager has no other power than its own deduction to find out the werewolves."
    , false
    , PowerState.PERMANENT
    , false
    , order
    , 6) {

    override fun clone() = Villager(this.order)
}